eksctl/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:edge as downloads
+FROM alpine:latest as downloads
```

Comment:
Does this image lock in a version?

Reply:
Certainly, It's better to fix it. I'll change it.